lib/camp.js

```diff
@@ -112,7 +112,7 @@ function jsonFromQuery(query, obj) {
 // This function updates ask.query with:
 // - application/x-www-form-urlencoded
 // - multipart/form-data
-function getQueries(ask, end) {
+function parseQuery(ask, end) {
```

Comment:
In the comment, you say "parse the query", and there seems to ever only be one "query".